src/App/Docs.hs

```diff
@@ -21,3 +22,6 @@ fossaYmlDocUrl = guidePathOf (maybe currentBranch ("v" <>) versionNumber) "/docs
 
 newIssueUrl :: Text
 newIssueUrl = sourceCodeUrl <> "/issues/new"
+
+strategyLangDocUrl :: Text -> Text
+strategyLangDocUrl path = guidePathOf (maybe currentBranch ("v" <>) versionNumber) ("/docs/references/strategies/languages/" <> path)
```

Comment:
`maybe currentBranch ("v" <>) versionNumber` should be a separate function in `App.Version`, I've seen it used quite a bit now.

Reply:
fixed in: https://github.com/fossas/fossa-cli/pull/805/commits/c64f7b9247ca7f0d676674640ce83ccdda5c2e34